bin/build-locales

```diff
@@ -39,5 +40,23 @@ poFiles.forEach((pofile) => {
     format: 'jed1.x',
     fuzzy: config.get('po2jsonFuzzyOutput'),
   });
-  fs.writeFileSync(jsonfile, json);
+
+  // Add the moment locale JS into our locale file, if one is available.
+  var defineLocale = null;
+  try {
+    const momentLocale = locale.replace('_', '-').toLowerCase();
+    defineLocale = new Function(
```

Comment:
Yeah, without doing this we end up writing `[object Object]`. Indeed though, it warrants an explanation.